code/game/gamemodes/gang/gang.dm

```diff
@@ -248,6 +249,19 @@ GLOBAL_LIST_INIT(gang_colors_pool, list("red","orange","yellow","green","blue","
 		gang_bosses += G.bosses
 	return gang_bosses
 
+/datum/game_mode/proc/shuttle_check()
+	if(forced_shuttle)
+		return
+	var/alive = 0
+	for(var/mob/living/L in GLOB.player_list)
```

Comment:
I'm aware, I can make the check more complex but I like simple. If a group of players are somehow surviving as ashwalkers then maybe the round should go on a little longer than it would otherwise.

My biggest concern is if CTF breaks it but thats already a pretty edge case and if the round is bad enough for an admin to CTF it should've been triggered already.

The only check I would surely add at this point is catatonic/braindead but I don't know the check for that and I'd like to see some averages and consider raising the % if that check is included.